public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -192,17 +192,28 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
 
             {hasLiveOption && (
               <LiveTailControls exploreId={exploreId}>
-                {(controls) => (
-                  <LiveTailButton
-                    splitted={splitted}
-                    isLive={isLive}
-                    isPaused={isPaused}
-                    start={controls.start}
-                    pause={controls.pause}
-                    resume={controls.resume}
-                    stop={controls.stop}
-                  />
-                )}
+                {(c) => {
+                  const controls = {
+                    ...c,
+                    start: () => {
```

Comment:
For everyone who goes back to this PR because of this code - this is the least impactful change we could do to have `datasourceType` in feature tracking. 😂😂